src/Features/LanguageServer/Protocol/Features/Options/DiagnosticOptionsStorage.cs

```diff
@@ -13,5 +13,8 @@ internal sealed class DiagnosticOptionsStorage
 
         public static readonly Option2<bool> LogTelemetryForBackgroundAnalyzerExecution = new(
             "dotnet_log_telemetry_for_background_analyzer_execution", defaultValue: false);
+
+        public static readonly Option2<bool> LightbulbSkipExecutingDeprioritizedAnalyzers = new(
+            "dotnet_lightbulb_skip_executing_deprioritized_analyzers", defaultValue: false);
```

Comment:
was this tested as well?  (do we think we even need this option at all?)

Reply:
Yes, a temporary feature flag that can be removed in future.